platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/platform_utils.py

```diff
@@ -18,5 +26,33 @@ def run_cmd(cmd):
         run_cmd(['mkdir', '-p', file_path])
     if not os.path.isfile(path):
         run_cmd(['touch', path])
-    if (mode is not None):    
+    if (mode is not None):
         run_cmd(['chmod', mode, path])
+
+def limit_execution_time(execution_time_secs: int):
```

Comment:
I'd expect the first param should be `func`
Please check

Reply:
I checked, and I see, for decorators with arguments it is recommended to use func as param in the inner function. If you don't mind